blocks/header/header.css

```diff
@@ -251,4 +630,606 @@ outline: none;
+    flex-direction: row;
+    align-items: center;
+    bottom: 20px;
+  }
+
+  header
+    nav
+    .nav-sections
+    .default-content-wrapper
+    > ul
+    > li[aria-expanded="true"]
+    > ul {
+    display: block;
+    position: absolute;
+    left: -1em;
+    width: 200px;
+    margin-top: 12px;
+    padding: 1em;
+    background-color: var(--light-color);
+    white-space: initial;
+  }
+
+  header .nav-menu-overlay .nav-menu-column.right .browse-country ul {
+    margin: 0;
+    padding: 0;
+    border: none;
+    margin-top: 30px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+
+  header .nav-menu-overlay .nav-menu-column.right .footer > ul {
+    gap: 0;
+    display: flex;
+    flex-direction: row;
+    border: none;
+    padding: 0;
+    margin: 0;
+    margin-left: 20px;
+    align-items: center;
+  }
+
+  header .nav-menu-overlay .nav-menu-column.right .submenu .footer > ul {
+    gap: 0;
+    display: flex;
+    flex-direction: row;
+    border: none;
+    padding: 0;
+    margin: 0;
+    margin-left: 20px;
+    align-items: center;
+  }
+
+  header .nav-menu-overlay .nav-menu-column.right .submenu .footer > ul > li {
+    color: var(--nav-menu-footer-color);
+    padding: 0 25px;
+    margin: 0;
+    border-right: 1px solid var(--nav-menu-footer-color);
+  }
+
+  header
+    .nav-menu-overlay
+    .nav-menu-column.right
+    .submenu
+    li:has(ul.browse-country)
+    ul {
+    margin: 0;
+    padding: 0;
+    border: none;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+
+  header
+    .nav-menu-overlay
+    .nav-menu-column.right
+    .submenu
+    li:has(ul.browse-country)
+    > ul {
+    border: none;
+  }
+
+  header
+    .nav-menu-overlay
+    .nav-menu-column.right
+    .submenu
+    li:has(ul.browse-country)
+    input[type="search"] {
+    min-width: 100%;
+  }
+
+  header nav[aria-expanded="true"] .nav-sections {
+    align-self: unset;
+  }
+
+  header nav[aria-expanded="true"] .nav-hamburger-icon {
+    left: 30px;
+  }
+
+  header nav .nav-sections .nav-drop::after {
+    content: "";
+    display: inline-block;
+    position: absolute;
+    top: 0.5em;
+    right: 2px;
+    transform: rotate(135deg);
+    width: 6px;
+    height: 6px;
+    border: 2px solid currentcolor;
+    border-radius: 0 1px 0 0;
+    border-width: 2px 2px 0 0;
+  }
+
+  header nav .nav-sections .nav-drop[aria-expanded="true"]::after {
+    top: unset;
+    bottom: 0.5em;
+    transform: rotate(315deg);
+  }
+
+  header nav .nav-sections .default-content-wrapper > ul > li > ul::before {
+    content: "";
+    position: absolute;
+    top: -8px;
+    left: 8px;
+    width: 0;
+    height: 0;
+    border-left: 8px solid var(--transparent);
+    border-right: 8px solid var(--transparent);
+    border-bottom: 8px solid var(--light-color);
+  }
+
+  header .nav-menu-overlay.open {
+    background-color: var(--transparent);
+    backdrop-filter: blur(2px);
```

Comment:
@padmashreeshetty123 I don't think this will work in Safari. Please check once.

Reply:
It is working. I checked

Reply:
as we discussed please fix this in a follow up PR.